template/cordova/lib/ConfigParser.js

```diff
@@ -73,6 +73,15 @@ WindowsConfigParser.prototype.getWindowsTargetVersion = function() {
     return preference;
 };
 
+WindowsConfigParser.prototype.getUAPTargetMinVersion = function() {
+    var preference = this.getPreference('uap-target-min-version');
+
+    if (!preference)
```

Comment:
Nit: code style: It just best to always add curlies for blocks - indentation based on editor tabs and other things can sometimes cause errors.